libbeat/processors/actions/docs/replace.asciidoc

```diff
@@ -11,15 +11,14 @@ contains a `field: field-name`, `pattern: regex-pattern` and
 `replacement: replacement-string`, where:
 
 * `field` is the original field name
-* `pattern` is regex pattern to match field's value
-* `replacement` is the replacement string to use for updating the field's value
+* `pattern` is the regex pattern to match the field's value
+* `replacement` is the replacement string to use to update the field's value
 
 The `replace` processor cannot be used to replace value with a completely new value. 
 
-TIP: You can replace field value to truncate part of field value or replace 
-it with a new string. It can also be used for masking PII information.
+TIP: The `replacement` field value can be used to truncate the `field` value or replace
```

Comment:
```suggestion
TIP: The `replacement` field value can be used to truncate the `field` value or replace
it with a new string. It can also be used for masking PII information.
```